include/deal.II/grid/grid_out.h

```diff
@@ -869,12 +869,14 @@ namespace GridOutFlags
     /**
      * Default constructor.
      */
-    Vtk(const bool output_cells    = true,
-        const bool output_faces    = true,
-        const bool output_co_faces = true)
+    Vtk(const bool output_cells         = true,
+        const bool output_faces         = true,
+        const bool output_co_faces      = true,
```

Comment:
I should have caught this earlier, but we don't generally use the term "co_faces". We just say "edges" and then say that this flag is simply ignored in 1d and 2d.

Reply:
This is how it is called internally... Should I nevertheless change it?

Reply:
Let's stick with our usual terminology, which is edge.